src/wallet/rpcnames.cpp

```diff
@@ -615,8 +627,13 @@ name_update ()
                           "there are already too many pending operations"
                           " on this name");
 
-    if (pendingOps > 0)
+    if (pendingOps > 0) {
       outp = mempool.lastNameOutput (name);
+      if (isDefaultVal) {
+        value = CNameScript(mempool.mapTx.find(outp.hash)->GetTx()
+          .vout[outp.n].scriptPubKey).getOpValue();
+      }
+    }
```

Comment:
Nit: Please use GNU style for the curly braces (and the same below on line 650).